fury/actors/tensor.py

```diff
@@ -0,0 +1,214 @@
+import os
+
+import numpy as np
+
+from fury import actor
+from fury.shaders import (attribute_to_actor, import_fury_shader,
+                          shader_to_actor, compose_shader)
+
+
+def tensor_ellipsoid(centers, axes, lengths, colors, scales, opacity):
```

Comment:
Will it still be named `tensor_ellipsoid` after the renaming decision?

Reply:
As the actor is named `ellipsoid` and I have to call the function in `actor.py`, I cannot use the same name for the function so I used `tensor_ellipsoid` instead.